src/receiver-ui.py

```diff
@@ -25,16 +28,35 @@ def receive():
     global continue_receiving
     continue_receiving = True
     while continue_receiving:
-        ip, port = receiver.consume_stream()
-        if ip and port:
-            subprocess.Popen(
-                [
-                    "ffplay",
-                    "-v",
-                    "warning",
-                    f"srt://{ip}:{port}?mode=listener",
-                ]
-            )
+        ip, port, is_rendezvous = receiver.consume_stream()
+        if ip and port and is_rendezvous:
+            time.sleep(3)
+            if is_rendezvous:
+                subprocess.Popen(
+                    [
+                        "ffplay",
+                        "-v",
+                        "warning",
+                        f"{UDP_SCHEME}://{LOCAL_HOST}:{INTERNAL_PORT}",
+                    ]
+                )
+
+                subprocess.Popen(
+                    [
+                        "srt-live-transmit",
```

Comment:
Since we've decided to run srt-live-transmit first in the sender, we should probably do the same in the receiver just for the sake of consistency